doc/mgmt/SONiC_OC_PortChannel_Interface_HLD.md

```diff
@@ -0,0 +1,439 @@
+          |        |     |  +--ro oc-ip:origin?          ip-address-origin
+          |        |     |  +--ro oc-ip:status?          enumeration
++         +--rw oc-eth:ethernet
++         |  +--rw oc-eth:config
+          |  |  +--rw oc-eth:mac-address?           oc-yang:mac-address
+          |  |  +--rw oc-eth:auto-negotiate?        boolean
+          |  |  +--rw oc-eth:duplex-mode?           enumeration
+          |  |  +--rw oc-eth:port-speed?            identityref
+          |  |  +--rw oc-eth:enable-flow-control?   boolean
++         |  |  +--rw oc-lag:aggregate-id?          -> /oc-if:interfaces/interface/name
++         |  +--ro oc-eth:state
+          |  |  +--ro oc-eth:mac-address?              oc-yang:mac-address
+          |  |  +--ro oc-eth:auto-negotiate?           boolean
+          |  |  +--ro oc-eth:duplex-mode?              enumeration
+          |  |  +--ro oc-eth:port-speed?               identityref
+          |  |  +--ro oc-eth:enable-flow-control?      boolean
+          |  |  +--ro oc-eth:hw-mac-address?           oc-yang:mac-address
+          |  |  +--ro oc-eth:negotiated-duplex-mode?   enumeration
+          |  |  +--ro oc-eth:negotiated-port-speed?    identityref
+          |  |  +--ro oc-eth:counters
+          |  |  |  +--ro oc-eth:in-mac-control-frames?    oc-yang:counter64
+          |  |  |  +--ro oc-eth:in-mac-pause-frames?      oc-yang:counter64
+          |  |  |  +--ro oc-eth:in-oversize-frames?       oc-yang:counter64
+          |  |  |  +--ro oc-eth:in-undersize-frames?      oc-yang:counter64
+          |  |  |  +--ro oc-eth:in-jabber-frames?         oc-yang:counter64
+          |  |  |  +--ro oc-eth:in-fragment-frames?       oc-yang:counter64
+          |  |  |  +--ro oc-eth:in-8021q-frames?          oc-yang:counter64
+          |  |  |  +--ro oc-eth:in-crc-errors?            oc-yang:counter64
+          |  |  |  +--ro oc-eth:in-block-errors?          oc-yang:counter64
+          |  |  |  +--ro oc-eth:out-mac-control-frames?   oc-yang:counter64
+          |  |  |  +--ro oc-eth:out-mac-pause-frames?     oc-yang:counter64
+          |  |  |  +--ro oc-eth:out-8021q-frames?         oc-yang:counter64
++         |  |  +--ro oc-lag:aggregate-id?             -> /oc-if:interfaces/interface/name
++         +--rw oc-lag:aggregation
++         |  +--rw oc-lag:config
+          |  |  +--rw oc-lag:lag-type?    aggregation-type
++         |  |  +--rw oc-lag:min-links?   uint16
++         |  +--ro oc-lag:state
+          |  |  +--ro oc-lag:lag-type?    aggregation-type
++         |  |  +--ro oc-lag:min-links?   uint16
+          |  |  +--ro oc-lag:lag-speed?   uint32
++         |  |  +--ro oc-lag:member*      oc-if:base-interface-ref
++         |  |  +--ro if-agmnt:fallback?  boolean  //Augmented
+
+```
+
+**Note:** openconfig-if-aggregate.yang data model is augmented to support LACP Fallback.
+
+### 3.6.2 CLI
+#### 3.6.2.1 Configuration Commands
+
+### Create a PortChannel
+`interface PortChannel <channel-number>`
```

Comment:
Need to provide the valid range support for <channel-number>

Reply:
Done!